app/models/finance/postpaid_billing_strategy.rb

```diff
@@ -6,7 +6,7 @@ class Finance::PostpaidBillingStrategy < Finance::BillingStrategy
   def daily(options = {})
     now = options[:now].presence || Time.now.utc
     bill_and_charge_each(options) do |buyer|
-      Rails.logger.info("Started to bill and charge buyer #{buyer.id} for provider #{buyer.provider_account_id} at #{now}")
+      Rails.logger.info("#{log_prefix(buyer)} started daily billing and charging at #{now} (postpaid)")
```

Comment:
I meant this, isn't it redundant to `Rails.logger.info("Finance::BillingStrategy.daily started for options #{options}")`?

And btw why some logs start with `[billing]` and some with `Finance::BillingStrategy.daily`.

Excuse the nitpicks :)

Reply:
It could be redundant, or could be not... 

In theory it is, because we call this method with a single buyer, but just in case it's not... (like multiple buyers for some reason are selected or things like that).

Regarding the second point, fair enough.... it's just that I chose `[billing] provider X buyer Y` as a common prefix, but a single `buyer` is not available in `Finance::BillingStrategy.daily`, so we couldn't apply that prefix there.

We could prefix with `[billing]` as well, but as you're saying it is very possible that this log in `Finance::BillingStrategy.daily` is redundant, so maybe we'll get rid of it.